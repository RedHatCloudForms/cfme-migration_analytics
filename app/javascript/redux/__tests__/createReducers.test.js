// TODO add unit tests for createReducers.js

it('works', () => expect(true).toBe(true));
