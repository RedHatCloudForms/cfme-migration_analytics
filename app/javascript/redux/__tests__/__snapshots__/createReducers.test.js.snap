// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create reducers produces a properly combined initial state 1`] = `
Object {
  "analytics": Object {
    "bundleStartResult": null,
    "bundleTask": null,
    "changeManifestResult": null,
    "detailedDataSelected": false,
    "errorChangingManifest": null,
    "errorDownloadingPayload": null,
    "errorFetchingBundleTask": null,
    "errorFetchingManifestInfo": null,
    "errorStartingBundle": null,
    "isChangingManifest": false,
    "isDownloadingPayload": false,
    "isFetchingBundleTask": false,
    "isFetchingManifestInfo": false,
    "isStartingBundle": false,
    "manifestInfo": null,
    "manifestUpdateModalVisible": false,
    "payloadFileName": null,
    "selectedProviders": Array [],
    "summaryData": null,
  },
  "providers": Object {
    "errorFetchingProviders": null,
    "isFetchingProviders": false,
    "providers": null,
  },
  "reports": Object {
    "errorFetchingReports": null,
    "errorFetchingResult": null,
    "errorFetchingTask": null,
    "errorRunningReport": null,
    "fetchingResultHrefs": Array [],
    "fetchingTaskHrefs": Array [],
    "numReportFetchesPending": 0,
    "reportRunsByHref": Object {},
    "reports": Array [],
    "resultsByHref": Object {},
    "runningReportHrefs": Array [],
    "tasksByHref": Object {},
  },
}
`;
